validator/sawtooth_validator/journal/consensus/poet1/poet_transaction_block.py

```diff
@@ -158,7 +165,7 @@ def update_block_weight(self, journal):
             assert self.wait_certificate
             self.aggregate_local_mean = self.wait_certificate.local_mean
 
-            if self.PreviousBlockID != NullIdentifier:
+            if self.PreviousBlockID != NULL_BLOCK_IDENTIFIER:
```

Comment:
It declared at the module level (similar to LOGGER), and pep8 specifies those follow all-caps.